packages/fx-core/resource/package.nls.json

```diff
@@ -250,34 +250,34 @@
+  "core.copilotPlugin.scaffold.summary.warning.swaggerVersion": "The OpenAPI description document is on Swagger version 2.0. No mitigation needed. It's converted to OpenAPI 3.0 and saved in \"%s\".",
+  "core.copilotPlugin.scaffold.summary.warning.teamsManifest.lengthExceeding": "\"%s\" cannot exceed %s characters. ",
+  "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingFullDescription": "Full description not found. ",
   "core.copilotPlugin.scaffold.summary.warning.teamsManifest.mitigation": "Mitigation: update \"%s\" field in \"%s\".",
-  "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCardTemlate": "Missing \"%s\" in command \"%s\".",
+  "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCardTemlate": "\"%s\" not found in command \"%s\".",
   "core.copilotPlugin.scaffold.summary.warning.teamsManifest.missingCardTemlate.mitigation": " Mitigation: create Adaptive Card template in \"%s\" and then update \"%s\" field to the relative path in \"%s\".",
-  "core.copilotPlugin.scaffold.summary.warning.api.optionalParametersOnly": "There is no required parameter defined in API \"%s\". The first optional parameter is set as the parameter for command \"%s\".",
-  "core.copilotPlugin.scaffold.summary.warning.api.optionalParametersOnly.mitigation": " Mitigation: If \"%s\" is not what you need, edit the parameter of command \"%s\" in \"%s\". The parameter name should match what's defined in \"%s\".",
+  "core.copilotPlugin.scaffold.summary.warning.api.optionalParametersOnly": "No required parameter defined in API \"%s\". The first optional parameter is set as the parameter for command \"%s\".",
+  "core.copilotPlugin.scaffold.summary.warning.api.optionalParametersOnly.mitigation": " Mitigation: If you don't need \"%s\", edit the \"%s\"command's parameter in  \"%s\" to match what's defined in \"%s\".",
   "core.createCapabilityQuestion.titleNew": "Capabilities",
   "core.createCapabilityQuestion.placeholder": "Select a capability",
   "core.createProjectQuestion.option.description.preview": "Preview",
   "core.createProjectQuestion.option.description.worksInOutlook": "Works in Teams and Outlook",
-  "core.createProjectQuestion.option.description.worksInOutlookM365": "Works in Teams, Outlook and the Microsoft 365 app",
-  "core.createProjectQuestion.option.description.worksInOutlookCopilot": "Works in Teams, Outlook and Copilot",
-  "core.createProjectQuestion.projectType.bot.detail": "Conversational or informative chat experiences that can automate repetitive tasks",
+  "core.createProjectQuestion.option.description.worksInOutlookM365": "Works in Teams, Outlook, and Microsoft 365 app",
+  "core.createProjectQuestion.option.description.worksInOutlookCopilot": "Works in Teams, Outlook, and Copilot",
+  "core.createProjectQuestion.projectType.bot.detail": "Create instant, engaging chat experiences that automate tasks seamlessly",
```

Comment:
People developing chatbot treat "conversational chat experience" as a standard experience. Not sure about the informative though, but I guess they really mean "conversational chat and notification experience"

Reply:
The logic behind the revised content is -
- Used term 'instant' to convey that the chat experiences provided by the bot happen promptly or without delay.
- Used term "engaging" to highlight the bot's ability to not only perform tasks but also to provide a positive and interactive experience, making it more appealing to users.

However, I get your point about conversational. So, will update the content.